src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/IOpenIdConnectEvents.cs

```diff
@@ -38,17 +28,22 @@ public interface IOpenIdConnectEvents : IRemoteAuthenticationEvents
         /// <summary>
         /// Invoked before redirecting to the identity provider to authenticate.
         /// </summary>
-        Task RedirectToAuthenticationEndpoint(RedirectContext context);
+        Task RedirectToIdentityProvider(RedirectContext context);
```

Comment:
lol, it's really nitpicking... though it's not the official name, "authentication endpoint" is not wrong since an OIDC request is both an authorization and an authentication request.

`RedirectToIdentityProvider` was renamed a few months because it was not clear enough. I'm not sure why you mentioned WS-Fed, but I guess you'd like to have a similar name for both protocols/middleware? Sounds like a bad idea, IMHO.

`RedirectToAuthorizationEndpoint` would be much better.